app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,7 @@
         android:theme="@style/Theme.Sobriety">
         <activity
             android:name=".Create"
+            android:windowSoftInputMode="adjustResize"
```

Comment:
This is necessary, to display the Snackbar "Can't create entries with duplicate keys" above the opened keyboard.